dashboard-api/.air.toml

```diff
@@ -0,0 +1,53 @@
+# Config file for [Air](https://github.com/cosmtrek/air) in TOML format
```

Comment:
Do we really need this file? I'd assume air comes with _safe_ and _sane_ defaults. 
Or could we remove the ones that comes by default?

Reply:
Yes but as you pointed we can remove most of the options. The ones that are required are cmd and bin.